docker-compose.yml

```diff
@@ -13,4 +13,4 @@ services:
       - "30120:30120"
       - "30120:30120/udp"
     environment:
-      LICENCE_KEY: "license-key-here"
+      LICENCE_KEY: "licence-key-here"
```

Comment:
Should this be `LICENSE_KEY` to match your changes made to the README?

Reply:
Er oops, good catch. Will push a fix.
You're on it :eyes: